app/experimenter/experiments/constants/nimbus.py

```diff
@@ -157,6 +157,8 @@ class NimbusTargetingConfig:
 
 TARGETING_STICKY = "experiment.slug in activeExperiments"
 
+TARGETING_HAS_PIN = "!doesAppNeedPin"
```

Comment:
Is something like this what you were thinking for the HAS_PIN targeting? I figure we don't actually need a full targeting config here, just a string constant.